assignments/assignment-5.adoc

```diff
@@ -102,22 +103,18 @@ second security group:
 
 === Create RDS database cluster
 
-Wordpress is a very popular Content Management System (CMS) and like most CMS
+WordPress is a very popular Content Management System (CMS) and like most CMS
 systems it requires a database to persist content and user account information.
 We will use a MariaDB database (MySQL opensource clone) managed by the RDS service to support this
 database requirement.
 
-Go to the RDS dashboard on AWS and launch a new MariaDB database cluster. The
-launch wizard may ask you if you plan to use the instance for production (multi-instance) or
-development purposes. In this case, select the production option (multi-instance). We want to deploy
-a primary and secondary database in multiple AZs.
+Go to the RDS dashboard on AWS and launch a new MariaDB database cluster.
 
-The MariaDB cluster should have the following properties:
+The MariaDB instance should have the following properties:
 
-  * DB Engine Version = ~10.2.21 (most any recent release should work)
-  * Template: Dev/Test
-  * DB Class = db.t2.micro
-  * Multi-AZ deployment: Yes
+  * DB Engine Version: ~10.2.21 (most any recent release should work)
+  * Template: Free tier
```

Comment:
I specifically require students to deploy a multi-AZ rds cluster in this assignment. The cost is about $0.95/day.

Reply:
If multi-AZ is required, the specification to use `db.t2.micro` does not seem to be possible, given that multi-AZ is only available for the `Production` and `Dev/Test` templates (for which the smallest available instance size is `db.m4.large`).

Reply:
Maybe you aren't setting the DB instance class to Burstable classes before making the instance selection.

Reply:
Yes, it looks like that's the culprit. I'll adjust the edits accordingly (and add a note about selecting `Burstable classes (includes t classes)`.